### PR TITLE
Update workspaces (screenshots + shared workspaces)

### PR DESCRIPTION
**Pull request by ujovlado**
Jira: https://keboola.atlassian.net/browse/DOCS-140

Changes:

- Updated all screenshots
- Added section about Shared workspaces

---

Ten text o shared workspaces je trosku prvoplanovy, tak pripadne dopln :)

### CODE REVIEW

**Comment by ujovlado** [comment]
Tak toto este budem malicko ladit, mame tam po novom Hibernate namiesto Terminate.

File: transformations/workspace/index.md [hunk 1]
```diff
@@ -134,6 +137,27 @@ or [File Output Mapping](/transformations/mappings/#file-output-mapping) (or bot
 Unloading data is useful, for example, when your ad-hoc analysis leads to
 valuable results, or when you trained a new model which you'd like to use in transformations.
 
+### Shared Workspaces
+
+Every member of a project can share his workspace with all project members. If you share workspace
+with other project members, then every member of a project can use your workspace, load data to it,
+restore it, etc.
```
Comment (odinuv) [inline transformations/workspace/index.md:144]: k tomu bych ale napsal neco v tom stylu, ze je to porad pod jednim uzivatelem, aby se lidi nedivili, az si budou prepisovat veci navzajem

**Comment by odinuv** [comment]
@ujovlado co s tímhle?

**Comment by ujovlado** [comment]
Zavrel by som to. Imho sa vacsina poriesila tu https://github.com/keboola/connection-docs/pull/333. A toto sa musi skontrolovat a ked tak pripravit nanovo.

**Pull request closed without merging**